.circleci/config.yml

```diff
@@ -147,7 +147,7 @@ jobs:
             - checkout
             - run:
                 name: Store path
-                command: echo 'export PATH=/usr/local/opt/flex/bin:/usr/local/opt/bison/bin:$PATH' >> $BASH_ENV
+                command: echo 'export PATH=/opt/homebrew/opt/flex/bin:/opt/homebrew/opt/bison/bin:$PATH' >> $BASH_ENV
```

Comment:
What is the reason for this?

Reply:
This is the path where homebrew places binaries on this new image.
I don't know why it changed, but it did.